examples/sensor_utils.ipynb

```diff
@@ -0,0 +1,316 @@
+{
```

Comment:
I'm not sure if we're still worried about posting full paths in notebooks, but there's a warning here that prints a full path.

Reply:
I may need to just remove the output of the notebook.